release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -473,6 +473,127 @@ The Local Storage Operator now has the ability to:
 [id="ocp-4-6-operators"]
 === Operators
 
+[id="ocp-4-6-operator-api"]
+==== Operator API now supported
+
+The Operator API introduced in {product-title} 4.5 as a Technology Preview
+feature is now supported and enabled by default. Previously, installing
+Operators using Operator Lifecycle Manager (OLM) required cluster administrators
+to be aware of multiple APIs, including CatalogSources, Subscriptions,
+ClusterServiceVersions, and InstallPlans. This single Operator API resource is a
+first step towards a more simplified experience discovering and managing the
+lifecycle of Operators in a {product-title} cluster.
+
+While previously only available in read-only mode, cluster administrators can
+now use the web console, in addition to the CLI, with this single API to
+interact with installed Operators. In addition, relevant OLM objects are now
+automatically labeled accordingly for the new Operator API for any Operators
+where the CSV is installed using a Subscription.
+
+Cluster administrators can discover and edit previously installed Operators
+using this API either using the web console or the CLI, for example using the
```

Comment:
@adellape I do not think cluster administrator can use the API with web console, and only the CLI. @njhale could you please confirm it? thanks

Reply:
OK, I was noticing in 4.6 that I could at least search for and view Operator resources, but that there was also a "Create Operator" button. But maybe it is still only read-only there?

Reply:
@adellape we can view the installed operators from web console, but it is not Operator API.  For Operator API to view status of resources of the installed operator, it is only CLI.

Reply:
OK thanks, I'll remove mention of web console here.

Reply:
see https://github.com/openshift/openshift-docs/pull/26309/files#r504136551